packages/analyticsdx-template-lint/test/unit/linter.test.ts

```diff
@@ -6,6 +6,7 @@
  */
 
 import { expect } from 'chai';
+import exp from 'constants';
```

Comment:
Nit: accidental import, I think. I end up getting this all the time, too. The linting might fail on this, I think.